assets/javascripts/src/actions.es6

```diff
@@ -28,6 +32,47 @@ export const GA_ENABLED = "GA_ENABLED";
 
 export const AUTOFILL = "AUTOFILL";
 
+export function openStripeCheckout(dispatch) {
+    const state = store.getState();
+    console.log('is this thing on');
+    let hide = () => {
+        dispatch({type:CLOSE_STRIPE});
+    };
+    let process = (token) => {
```

Comment:
should be a `const` rather than `let`. 
i'd probably define it inside the stripe module and import it rather than inside this function